content/en/docs/aws/rds.md

```diff
@@ -0,0 +1,92 @@
+
+# Below command will retrieve list of Private subnets
+# You need to use at least two Subnets from your List
+aws ec2 describe-subnets --filters Name=tag:alpha.eksctl.io/cluster-name,Values=$AWS_CLUSTER_NAME Name=tag:aws:cloudformation:logical-id,Values=SubnetPrivate* | jq -r '.Subnets[].SubnetId'
+
+# Below command will retrieve SecurityGroupId for your Worker nodes
+# This assumes all your Worker nodes share same SecurityGroups
+INSTANCE_ID=$(aws ec2 describe-instances --query 'Reservations[*].Instances[*].InstanceId' --filters "Name=tag-key,Values=eks:cluster-name" "Name=tag-value,Values=$AWS_CLUSTER_NAME" --output text)
+for i in "${INSTANCE_IDS[@]}"
+do
+  echo "SecurityGroup for EC2 instance $i ..."
+aws ec2 describe-instances --instance-ids $i | jq -r '.Reservations[].Instances[].SecurityGroups[].GroupId'
+done  
+```
+You can either use console or use attached CloudFormation template to deploy Amazon RDS database
+
+{{% alert title="Warning" color="warning" %}}
+The CloudFormation template deploys Amazon RDS for MySQL that is intended for Dev/Test environment.
+We highly recommend deploying Multi-AZ database for Production. Please review RDS [documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Welcome.html) to learn more
+{{% /alert %}}
+
+[{{<figure src="/docs/images/aws/cloudformation-launch-stack.png">}}](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=kubeflow-db&templateURL=https://cloudformation-kubeflow.s3-us-west-2.amazonaws.com/rds.yaml)
+
+Remember to select correct **region** for your cluster and click Next. We recommend you to change the **DBPassword**, if not it will dafault to `Kubefl0w`. Select VpcId, Subnets and SecurityGroupId before clicking Next. Take rest all defaults and click **Create Stack**. Once the CloudFormation is completed, click on Outputs tab to get RDS endpoint. We will use it in next step while installing Kubeflow   
+
+![dashboard](/docs/images/aws/cloudformation-rds-output.png)
+
+### Deploy Kubeflow Pipeline and Metadata using Amazon RDS
+
+1. Follow the [install documentation](https://www.kubeflow.org/docs/aws/deploy/install-kubeflow/) until [Deploy Kubeflow](https://www.kubeflow.org/docs/aws/deploy/install-kubeflow/#deploy-kubeflow) section.
+Modify `${CONFIG_FILE}` file to add `external-mysql` in both pipeline and metadata kustomizeConfigs and remove mysql database as shown below.  
+
+    ![dashboard](/docs/images/aws/external-mysql-rds.png)
+
+2. Run the follow commands to build Kubeflow installation:
+
+    ```
+    cd ${KF_DIR}
+    kfctl build -V -f ${CONFIG_FILE}
+    ```
+    This will create two folders `aws_config` and `kustomize` in your environment. Edit `params.env` file for the external-mysql pipeline service (`kustomize/api-service/overlays/external-mysql/params.env`) and update values based on your configuration
+
+    ```
+    mysqlHost=<$RDSEndpoint>
```

Comment:
Do we have some guidance for users to retrieve RDSEndpoint?

Reply:
Its already there for CloudFormation users, I just added comment for non-cloudformation users as well